tests/main.sh

```diff
@@ -31,22 +38,22 @@ IMAGE=$(echo $REF | awk -F':' '{print $1}')
 SHELL_CMD_TEMPLATE="docker run --rm -i \$SHELL_CMD_FLAGS $REF \
 	$([ ${DISTRO} == 'debian' ] && echo bash || echo sh) -c"
 # determine reference size
-if [ $DISTRO == alpine ] && [ $PY_VER == '3.9' ]; then
-	SIZE_LIMIT=702
+if [ $DISTRO == alpine ] && [ $PY_VER == '3.10' ]; then
+	SIZE_LIMIT=991
+elif [ $DISTRO == alpine ] && [ $PY_VER == '3.9' ]; then
+	SIZE_LIMIT=723
 elif [ $DISTRO == alpine ] && [ $PY_VER == '3.8' ]; then
-	SIZE_LIMIT=661
+	SIZE_LIMIT=682
 elif [ $DISTRO == alpine ] && [ $PY_VER == '3.7' ]; then
-	SIZE_LIMIT=669
-elif [ $DISTRO == alpine ] && [ $PY_VER == '3.6' ]; then
-	SIZE_LIMIT=595
+	SIZE_LIMIT=693
+elif [ $DISTRO == debian ] && [ $PY_VER == '3.10' ]; then
+	SIZE_LIMIT=1210
 elif [ $DISTRO == debian ] && [ $PY_VER == '3.9' ]; then
-	SIZE_LIMIT=940
+	SIZE_LIMIT=962
 elif [ $DISTRO == debian ] && [ $PY_VER == '3.8' ]; then
-	SIZE_LIMIT=893
+	SIZE_LIMIT=915
 elif [ $DISTRO == debian ] && [ $PY_VER == '3.7' ]; then
-	SIZE_LIMIT=896
-elif [ $DISTRO == debian ] && [ $PY_VER == '3.6' ]; then
-	SIZE_LIMIT=822
+	SIZE_LIMIT=912
```

Comment:
Do you know what is the reason for the `alpine` image increase? I thought our previous effort in `miniconda3` only affected `debian`. I would have expected a lower footprint overall since we removed `inotifytools`. `otumat` was added but it should have already been available since it is a `datajoint` dependency (which are already installed). Maybe if `otumat` was upgraded with more features?

Reply:
I'm not sure why but it seems that `inotify` is still installed despite being removed from the `_requirements.txt` files. I tried removing otumat from the `pip_requirements.txt` but the image size does not change.

Reply:
@guzman-raphael I found something interesting. Despite the size of alpine builds not increasing in the new `djlab`, `djbase`, or `miniconda3` images, updating the base image hash in `djlabhub` causes a ~21 MB increase in size for the alpine images.